wikipedia-dark.user.css

```diff
@@ -163,6 +163,7 @@
   tr[style*="background-color: #f5faff;" i],
   tr[style*="background:#E9E9E9" i] > *,
   table.infobox.hproduct > tbody > tr[style*="background: #F0F0F0;" i],
```

Comment:
Because wikipedia can be edited by anyone, there is no consistency in styling... I don't remember the exact reason for that selector choice, but I'm guessing there was a reason. But go ahead and remove  the `> *` and we'll keep an eye out for any missing styles.

Reply:
Sure I'll remove that but I was more specifically referring to `table.infobox.hproduct > tbody >` in line 165. Is that part necessary?

Reply:
Leave it for now, we can fix it later... It looks it was originally added in #7, altered in https://github.com/StylishThemes/Wikipedia-Dark/commit/c365792cb49d843a5f75f698b25f09d219549217; but I think wikipedia altered the box again, because even removing that line doesn't change the appearance. What do you think about removing the `table.infobox.hproduct > tbody >` @Poorchop?

Reply:
Sorry for the delayed response, I'm up to my eyes in work. I remember that commit but as you already mentioned, Wikipedia can be edited by anyone and there's no consistency so who even knows if that rule is applicable anymore. I think it's best to spend some time with the modified style or maybe push it to a dev branch until we're reasonably sure that nothing breaks from deleting a rule.

Reply:
Alrighty! I managed to pull out the old style from the [infobox history](https://en.wikipedia.org/w/index.php?title=Template:Infobox_video_game&oldid=674824106). The incorrect colours are still perfectly fixed without `table.infobox.hproduct > tbody >`. So I think it should be okay to drop that part. Let me know your thought!